packages/enzyme-test-suite/test/ShallowWrapper-spec.jsx

```diff
@@ -1716,7 +1716,7 @@ describe('shallow', () => {
 
       it('works without memoizing', () => {
         const wrapper = shallow(<RendersApp />);
-        expect(wrapper.debug()).to.equal('<App />');
+        expect(wrapper.debug()).to.equal(is('>= 17') ? '<AppMemoized />' : '<App />');
```

Comment:
This appears to be due to an apparent change in how `React.memo()` operates in React 17. Prior to 17 is seemed to return a copy of the component. In 17 it seems to return a reference.